pkg/controller/constraint/constraint_controller.go

```diff
@@ -182,14 +183,16 @@ func (r *ReconcileConstraint) Reconcile(request reconcile.Request) (reconcile.Re
 		if err = util.SetHAStatus(instance, status); err != nil {
 			return reconcile.Result{}, err
 		}
-		if err := r.cacheConstraint(instance); err != nil {
-			r.constraintsCache.addConstraintKey(constraintKey, tags{
-				enforcementAction: enforcementAction,
-				status:            metrics.ErrorStatus,
-			})
-			reportMetrics = true
+		if c, err := r.opa.GetConstraint(context.TODO(), instance); err != nil || !constraints.SemanticEqual(instance, c) {
```

Comment:
Was just looking at framework, and noticed this: https://github.com/open-policy-agent/frameworks/blob/master/constraint/pkg/core/constraints/constraints.go#L18 can you PTAL https://github.com/open-policy-agent/frameworks/pull/73

Reply:
That should be fixed now. LGTY?

Reply:
@ritazh ping

Reply:
Can you pls rebase?

Reply:
rebased